src/query/procedure/module.cpp

```diff
@@ -715,7 +715,7 @@ bool SharedLibraryModule::Load(const std::filesystem::path &file_path) {
   spdlog::info("Loading module {}...", file_path);
   file_path_ = file_path;
   dlerror();  // Clear any existing error.
-  handle_ = dlopen(file_path.c_str(), RTLD_NOW | RTLD_LOCAL);
+  handle_ = dlopen(file_path.c_str(), RTLD_NOW | RTLD_LOCAL | RTLD_DEEPBIND);
```

Comment:
Can you use `SharedLibraryHandle` also here?

Reply:
Hmm I wouldn't since the library that's been explicitly loaded and also explicitly closed in `Close` method, and that's been handled by `SharedLibraryModule`

Reply:
Makes sense 😄